docs/ballooning.md

```diff
@@ -23,10 +23,14 @@ The device can be configured with the following options:
   allocate some memory which would make the guest enter an out-of-memory state,
   the kernel will take some pages from the balloon and give them to said
   process instead asking the OOM killer process to kill some processes to free
-  memory. Note that this applies to allocations from guest processes which would
-  make the system enter an OOM state. This does not apply to instances when the
-  kernel needs memory for its activities (i.e. constructing caches), or when the
-  user requests more memory than the amount available through an inflate.
+  memory. Note that this applies to physical page allocations in the kernel
+  which belong to guest processes. This does not apply to instances when the
+  kernel needs memory for its activities (i.e. constructing caches), when the
+  user requests more memory than the amount available through an inflate, or
```

Comment:
should it be
```suggestion
  user requests more memory than the amount available through a deflate, or
```
?

or clearer (from my pov):
```
user requests more memory than the currently _held_ by balloon, or
```
or
```
user requests more memory than the currently available to the balloon for releasing, or
```

Reply:
I'll go with the second one.